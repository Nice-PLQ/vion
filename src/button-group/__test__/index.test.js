import { mount } from '@vue/test-utils';
import VButton from '../../button/button.vue';
import VButtonGroup from '../button-group.vue';

describe('ButtonGroup', () => {
  const Component = {
    template: `
      <v-button-group>
        <v-button type="primary" size="large">主按钮</v-button>
        <v-button size="large">默认按钮</v-button>
      </v-button-group>
    `
  };
  test('按钮组', () => {
    const wrapper = mount(Component, {
      global: {
        components: { VButton, VButtonGroup }
      }
    });
    expect(wrapper.element).toMatchSnapshot();
    expect(wrapper.findAllComponents('.v-button').length).toBe(2);
  });
});
