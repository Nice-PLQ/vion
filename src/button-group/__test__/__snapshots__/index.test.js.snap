// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup 按钮组 1`] = `
<div
  class="v-button-group"
>
  
  <button
    class="v-button v-button_primary v-button_large"
  >
    <div
      class="v-button__inner"
    >
      <!--v-if-->
      
      主按钮
      
    </div>
  </button>
  <button
    class="v-button v-button_default v-button_large"
  >
    <div
      class="v-button__inner"
    >
      <!--v-if-->
      
      默认按钮
      
    </div>
  </button>
  
</div>
`;
